src/App.js

```diff
@@ -17,15 +17,29 @@ Amplify.configure({
   }
 });
 
-const MyApp = (props) => {
-  const token = props?.authData?.signInUserSession?.idToken?.jwtToken;
-  setToken(token)
+// TO-DO: figure out how to integrate DHIS2 i18n with amplify
+I18n.putVocabulariesForLanguage('en', {
+  'Sign In': 'Login', // Tab header
+  'Sign in': 'Login', // Button label
+  Password: 'Enter your password', // Password label
+  'Forgot your password?': 'Reset Password',
+});
```

Comment:
I think it would be good to have some sort of indication that this is a Crosscut login (instead of a DHIS2 login). Maybe `Crosscut Sign In`?